OptimizelySDK/OptlyConfig/OptimizelyConfigService.cs

```diff
@@ -279,7 +277,7 @@ private string GetExperimentAudiences(Experiment experiment, ProjectConfig proje
             return GetSerializedAudiences(s, projectConfig.AudienceIdMap);
         }
 
-        readonly string[] AUDIENCE_CONDITIONS = { "and", "or", "not" };
+        private readonly string[] AUDIENCE_CONDITIONS = { "and", "or", "not" };
```

Comment:
I think these can be const.

Reply:
Changing to const actually causes the allocation issue (IE its not a const evaluation instead it has to be interpreted)

I've left it as private readonly for now